javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,149 @@
+/*=============================================
+                WARM UP
+=============================================*/
+
+//**************************************************************************************
+// 1. Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+//**************************************************************************************
+
+setTimeout(() => {
+  console.log(`Call after 2.5 seconds`);
+}, 2500);
+
+//**************************************************************************************
+// 2. Create a function that takes 2 parameters: delay and stringToLog.
+//    Calling this function should log out the stringToLog after delay seconds.
+//    Call the function you have created with some different arguments.
+//**************************************************************************************
+
+function ex2(delay, stringToLog) {
+  let howlong = delay * 1000;
+  setTimeout(() => {
+    console.log(stringToLog);
+  }, howlong);
+}
+
+ex2(5, "This string logged after 5 seconds");
+ex2(3, "This string logged after 3 seconds");
+
+//**************************************************************************************
+// 3. Create a button in html.
+//    When clicking this button, use the function you created in the previous task
+//   to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+//**************************************************************************************
+const button1 = document.getElementById("ex3");
+
+button1.addEventListener("click", ex3);
+
+function ex3() {
+  ex2(5, "Called after 5 seconds");
+}
```

Comment:
Here you could use function ex2 directly in addEventListener.
Like:
button1.addEventListener("click", ex2(5, "Called after 5 seconds"));